src/core/lib/transport/call_spine.h

```diff
@@ -450,7 +441,10 @@ auto OutgoingMessages(CallHalf h) {
 
 // Forward a call from `call_handler` to `call_initiator` (with initial metadata
 // `client_initial_metadata`)
-void ForwardCall(CallHandler call_handler, CallInitiator call_initiator);
+void ForwardCall(
+    CallHandler call_handler, CallInitiator call_initiator,
+    absl::AnyInvocable<void(ServerMetadata&)>
+        on_server_trailing_metadata_from_initiator = [](ServerMetadata&) {});
```

Comment:
Please document this new parameter.

Reply:
Also, can you please add a test for this parameter?

Reply:
done